waterlog/src/Pages/Home.js

```diff
@@ -1,19 +1,19 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import Wastage from './../Components/Wastage';
 import AlertsMap from './../Components/AlertsMap';
 import TankLevels from './../Components/TankLevels';
 
 class HomeComponent extends Component {
-  render() {
-    return (
-      <div>
-        <p className="home-text">Alerts Map</p>
-      <AlertsMap />
-      <Wastage />
-      <p className="home-text">Tank Levels</p>
-      <TankLevels />
-      </div>
-    );
-  }
+	render() {
```

Comment:
What up with the tab spacing ?

Reply:
we are using 2 spaces

Reply:
Someone recommended that we use 2 spaces instead of 4